README.md

```diff
@@ -158,3 +181,31 @@ Some tests may require an EasyPost user with a particular set of enabled feature
 - `USPS_CARRIER_ACCOUNT_ID` (eg: one-call buying a shipment for non-EasyPost employees)
 - `PARTNER_USER_PROD_API_KEY` (eg: creating a referral user)
 - `REFERRAL_CUSTOMER_PROD_API_KEY` (eg: adding a credit card to a referral user)
+
+#### Mocking
+
+We have implemented a custom, lightweight HTTP mocking functionality in this library that allows us to mock HTTP calls and responses.
```

Comment:
nit: the we/us bit in here isn't necessary. It may read better if stated "there is a lightweight"...